third_party/2/six/__init__.pyi

```diff
@@ -27,7 +27,7 @@ PY2 = True
 PY3 = False
 PY34 = False
 
-string_types = basestring,
+string_types = Union[str, unicode],
```

Comment:
I think the trailing comma here is fooling you. It might work though if you wrote `string_types = (str, unicode)`.